### PR TITLE
[#1996] Get wrappers loading necessary for Popcorn to use converted play...

### PR DESCRIPTION
**Pull request by humphd**
...ers

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @humphd received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/09ee82363f8ca46/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/09ee82363f8ca46/output.txt

Total script time: 0.63 mins

Server is running on http://pr795.butyr.org/templates/basic/
Run the Test Suite: http://pr795.butyr.org/test/

File: src/core/page.js [hunk 1]
```diff
@@ -38,7 +37,49 @@ define( [ "core/logger", "core/eventmanager" ], function( Logger, EventManagerWr
...
+            return !!window.Popcorn && !!window.Popcorn._MediaElementProto;
+          }
+        },
+
+        {
+          type: "js",
+          url: "{popcorn-js}/wrappers/html5/popcorn.HTMLMediaElement.js",
+          check: function(){
+            return !!window.Popcorn &&
+                   !!window.Popcorn.HTMLVideoElement &&
```
Comment (jbuck) [inline src/core/page.js:57]: Do you need to add `HTMLMediaElement` as well?

Comment (humphd) [inline src/core/page.js:57]: No, this is right.  The file is just called that, but there is only HTMLVideoElement and HTMLAudioElement.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/2cba60aa96c6f8e/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/2cba60aa96c6f8e/output.txt

Total script time: 0.74 mins

Server is running on http://pr795.butyr.org/templates/basic/
Run the Test Suite: http://pr795.butyr.org/test/

**Pull request merged**